Docs/General/ElectronicsCrashCourse.md

```diff
@@ -76,10 +80,21 @@ When you start up the robot you may see red error messages in the drive station
 If you see that, something might be broken on the CAN network.
 
 We have to set the IDs of the devices on the CAN network to make sure they are all uniquely and correctly identified.
-It would be very awkward to try to drive the robot and accidentally run the elevator!
+It would be very awkward to try to drive the robot and accidentally run the elevator instead of the drivetrain!
 To set the IDs of devices on the CAN network we use the [Tuner X](https://pro.docs.ctr-electronics.com/en/stable/docs/tuner/index.html) app built by CTRE.
 This app also has features to test and configure devices on the CAN network.
 
+### CANivore
+
+![CANivore](../../Assets/CANivore.webp)
```

Comment:
this is not that important but could you make the images a bit smaller or at least around the same size

Reply:
idk if i can do formatting let me take a look